src/main/java/io/candydoc/ddd/tactical_pattern/Factory.java

```diff
@@ -0,0 +1,30 @@
+package io.candydoc.ddd.tactical_pattern;
+
+import io.candydoc.ddd.model.DDDPattern;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class Factory extends TacticalModel {
+
+  private static final Set<DDDPattern> ALLOWED_DEPENDENCIES =
```

Comment:
A factory can also create value objects and domain entities but *must not* depend on domain service *nor* repositories.

The factory should have its own PR.